init.sh

```diff
@@ -84,6 +110,22 @@ fi
 echo "Copying contents to the installation directory..."
 cp -R "$temp_directory"/sp-files/* "$temp_directory"/sp-files/.??* "$install_directory"
 
+# Delete files with unpreferred filetype in the installation directory
+if [ -z "${default_filetype_choice:-}" ]; then
+    echo "Default filetype not defined, so proceed with deleting unpreferred filetype."
+    if [ "$file_type" = 'md' ]; then
+        echo "Deleting rst files..."
+        rm "$install_directory"/doc-cheat-sheet.rst
+        rm "$install_directory"/index.rst
```

Comment:
Thanks @akcano !

I was conflicted about using a wildcard here for one specific reason (maybe too specific):

If someone has written a readme.md to get started, then runs `init.sh .` to target the current working directory, then opts for `rst` as the filetype, this would delete their original file. A worse case would be someone who (for one reason or another) decides to run `init.sh` with default args in a directory with a set of markdown files.

In the original implementation only the md/rst files that need to be generated by the starter pack would be removed.

As `init.sh` would be run within a git repo it would probably be frustrating rather than fatal (the changes could be reverted).

What do you think?